backend/src/kitconcept/intranet/profiles/default/types/Document.xml

```diff
@@ -11,7 +11,7 @@
 
     <element value="kitconcept.intranet.location" />
     <element value="kitconcept.intranet.organisational_unit" />
-    <element value="kitconcept.intranet.lcm" />
+    <element value="kitconcept.intranet.clm" />
```

Comment:
I forgot this was already enabled.

Upgrading it like this will add the new behavior but not remove the old one (because there is purge="false" on line 9 and we are not specifying the full list of behaviors).

We can do this:

```suggestion
    <element value="kitconcept.intranet.lcm" remove="true" />
    <element value="kitconcept.intranet.clm" />
```